gpg_keys.dd

```diff
@@ -3,16 +3,26 @@ Ddoc
 $(P Since 2.067.0 dmd binaries and git tags are signed by D's release managers. The following GnuPG keys can be used to verify those signatures.)
 
 $(PRE
-pub   4096R/0xAB8FE924C2F7E724 2014-09-01 [expires: 2017-02-26]
+pub   4096R/0xAB8FE924C2F7E724 2014-09-01 [expires: 2020-03-25]
       Key fingerprint = AFC7 DB45 693D 62BB 472B  F27B AB8F E924 C2F7 E724
+uid                            Martin Nowak (dawg) $(LT)me@dawg.eu$(GT)
+uid                            Martin Nowak $(LT)martin.nowak@plugintheworld.com$(GT)
 uid                            Martin Nowak $(LT)code@dawg.eu$(GT)
-sub   4096R/0xB273811612BB1939 2015-02-27 [expires: 2018-03-03]
-sub   4096R/0xA78068C444E12E4D 2014-09-01 [expires: 2018-03-03]
+uid                            Martin Nowak $(LT)martin@dlang.org$(GT)
+sub   4096R/0xB273811612BB1939 2015-02-27 [expires: 2020-03-25]
+sub   4096R/0xA78068C444E12E4D 2014-09-01 [expires: 2020-03-25]
 
-pub   4096R/0xC13C70B87267B52D 2012-03-20 [expires: 2016-03-19]
-      Key fingerprint = 0B96 609E 270F 565C 1329  2B24 C13C 70B8 7267 B52D
-uid                            David Soria Parra $(LT)dsp@php.net$(GT)
-sub   4096R/0x4A208C733953AF82 2012-03-20 [expires: 2016-03-19]
+pub   4096R/0x2055F76601A36FB0 2015-11-24 [expires: 2020-11-22]
+      Key fingerprint = 8FDB 8D35 7AF4 68A9 428A  CE3C 2055 F766 01A3 6FB0
+uid                            Sebastian Wilzbach $(LT)sebi@wilzbach.me$(GT)
+uid                            Sebastian Wilzbach $(LT)seb@wilzba.ch$(GT)
+sub   4096R/0xFEB1F8D40328010D 2015-11-24 [expires: 2020-11-22]
```

Comment:
FYI: I updated the expiration of my key a few days ago as I didn't want to experience the same breakage as you did.
So while you are at updating our keyring, you might use the fingerprint too:

http://pgp.mit.edu/pks/lookup?op=vindex&search=0x2055F76601A36FB0
https://pgp.wilzba.ch/seb.asc